soliket/lensing/lensing.py

```diff
@@ -126,12 +124,32 @@ def get_requirements(self):
             }
         }
 
+
+    
     def _get_data(self):
         import sacc
-
         s = sacc.Sacc.load_fits(self.datapath)
```

Comment:
This could probably be refactored so the `load_fits` only needs to happen once, and the sacc gets stored as e.g. `self.sacc_data`